sources/platform/integrations/pinecone.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Pinecone
+description: Learn how to integrate Apify with Pinecone in order to feed data crawled from the web into the Pinecone vector database.
+sidebar_position: 11.22
+slug: /integrations/pinecone
+---
+
+**Learn how to integrate Apify with Pinecone to feed data crawled from the web into the Pinecone vector database.**
+
+---
+
+> For more information on Pinecone, visit its [documentation](https://docs.pinecone.io).
+
+## What is Pinecone?
+
+Pinecone is a managed vector database that allows users to store and query dense vectors for AI applications such as recommendation systems, semantic search, and retrieval augmented generation.
+
+## How to integrate Apify with Pinecone?
+
+This example focuses on integrating Apify with Pinecone using the Apify Actor.
+
+Before we start, ensure you have a [Pinecone database](https://www.pinecone.io/) set up.
+Create a Pinecone index and obtain the Pinecone API token.
+Additionally, you need an [OpenAI API key](https://openai.com/index/openai-api/) and [Apify API token](https://docs.apify.com/platform/integrations/api#api-token).
+
+Next, install the apify-client to interact with the Apify API:
```

Comment:
Is there a way to set this up manually in UI, using the actor-to-actor integration? If yes, please can we explain it here? e.g. have few sections how to set it up, one in Python, second in UI, another (in the future) with JS.

Reply:
Thanks. Good point. I added the UI section with screenshots.